README.markdown

```diff
@@ -898,6 +898,18 @@ resource.request().onComplete { [weak self] response in
 }
 ```
 
+As a *better* alternative to the weak self/strong self scenario, if we just need to capture some variables or objects, instead of addressing the whole self,we can use a capture list with the name of the vars needed.
```

Comment:
There is a small typo `self,we` should have a space

Reply:
Fixed!